records_versioning/migrations/0001_initial.py

```
@@ -0,0 +1,44 @@
+            name='RecordVersion',
+            fields=[
+                ('id', models.BigAutoField(primary_key=True, serialize=False)),
+                ('resource_type', models.CharField(max_length=48)),
+                ('resource_id', models.UUIDField()),
+                ('version', models.PositiveIntegerField()),
+                ('prev_version', models.PositiveIntegerField(blank=True, null=True)),
+                ('snapshot', models.JSONField()),
+                ('diff', models.JSONField(blank=True, null=True)),
+                ('meta', models.JSONField(blank=True, default=dict)),
+                ('changed_at', models.DateTimeField(default=django.utils.timezone.now)),
+                ('changed_by', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.PROTECT, to=settings.AUTH_USER_MODEL)),
```

Ruff (0.12.2)

Line too long (147 > 88)

(E501)